docs/Developers/Newcomers_Guide.rst

```diff
@@ -131,20 +131,13 @@ Step 3. Picking Up an Issue
 Now it is time to pick an issue.
 Here is the link that will lead you to
 `newcomer issues <https://coala.io/new>`_.
-
-.. note::
-
-    You need to be logged in before you follow the Newcomers issues link.
-
-.. seealso::
-
-    For more information about what bears are, please check the following link:
-    `writing native bears <coala.io/writingbears>`_
-
+(You need to be logged in before you follow the Newcomers issues link.)
```

Comment:
I would keep the old `note::`, imo such `(blablabla)` constructs on their own without normal sentences which serve some context a note looks and integrates better into reading flow^^